wikimedia/wikipedia-ios
# Article as a Living Doc: Add external link icon to modal

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T266256

### Notes
* Adds in the external link icon to AaaLD modal.

### Test Steps
1. May need code from this PR to test: https://github.com/wikimedia/wikipedia-ios/pull/3763
1. Open Article as a Living Doc modal. See journal, news, and website citations, ensure they have working links with an external link icon. 

### Screenshots/Videos *(if applicable)*
<img width="409" alt="Screenshot 4" src="https://user-images.githubusercontent.com/9295855/98611338-997db500-22a6-11eb-88b9-5dd06d34b4a9.png">
